package.json

```diff
@@ -46,7 +48,7 @@
     "@babel/plugin-proposal-class-properties": "^7.14.5",
     "@testing-library/jest-dom": "^4.2.4",
     "@wordpress/block-editor": "3.1.0",
-    "@wordpress/block-library": "2.8.0",
+    "@wordpress/block-library": "^8.25.0",
```

Comment:
Why this change?

Reply:
Updated to get all the new controls provided by WP Block Library